experiments/AMIP/coupler_driver.jl

```diff
@@ -858,20 +858,22 @@ if ClimaComms.iamroot(comms_ctx)
         # Compare against observations
         if t_end > 84600
             @info "Error against observations"
+            output_dates = cs.dates.date0[] .+ Second.(atmos_sim.integrator.sol.t)
```

Comment:
It would be good to add a comment that `dt_save_to_sol` needs to be consistent with the required averaging. For most of our runs `dt_save_to_sol` is `10days` but the seasonal average using this method would require `1days` (for daily averaged data).